control/control.rnc

```diff
@@ -566,9 +566,14 @@ system_role = element system_role {
     ## texts section: ${id} and ${id}_description.
     element id { text } &
 
-    ## option to not preselect any role, by default first one is preselected
+    ## option to not preselect any role, by default first one by order is preselected
```

Comment:
suggested change to clarify this a bit:

"by default, **the** first one **(by sort order, see below)** is preselected"

Reply:
make sense, will add it